onnx_chainer/testing/test_onnxruntime.py

```diff
@@ -23,6 +23,53 @@
 TEST_OUT_DIR = 'out'
 
 
+def check(model, args, name, opset_version):
+    test_path = gen_test_data_set(model, args, name, opset_version)
+    check_model_expect(test_path)
+
+
+def gen_test_data_set(model, args, name, opset_version):
+    model.xp.random.seed(42)
+    test_path = os.path.join(
```

Comment:
Just a note: I don't think we should do something immediately, but I guess now we output a few giga bytes to "out" directory? Perhaps it's better to auto-remove some ONNX models generated by test_rt.py. For example, I'd keep large ONNX models only for the most recent opset_version.

Reply:
agree, pytest framework support command args, for example, add `pytest --keep-models=False` option easily (maybe), I'll survey these options. Thanks.